orderer/common/cluster/clusterservice.go

```diff
@@ -235,11 +253,11 @@ func (c *ClusterService) ConfigureNodeCerts(channel string, newNodes []NodeIdent
 		c.Logger.Debugf("Refreshing the %s channel nodes identities", channel)
 		delete(c.MembershipByChannel, channel)
 	}
-	channelCert = make(map[uint64]*msp.SerializedIdentity)
+	channelCert = &ChannelMembersConfig{}
+	atomic.StoreInt64(&channelCert.LastUpdatedTime, time.Now().UnixNano())
```

Comment:
won't this technique close all streams connected?

Reply:
No. it will be closing the streams only when the next msg arrived based on the lastUpdatedTime and streamCreation.

**clusterservice.go ... func handleMessage**
``` 
	if streamRefTime != atomic.LoadInt64(&s.MembershipByChannel[channel].LastUpdatedTime) {
		s.Logger.Debugf("Channel %s stream is stale", channel)
		return errors.New("stream is stale")
	}
```